app/(map)/_api/useToggleLikeProfile.ts

```diff
@@ -0,0 +1,52 @@
+import { toast } from "sonner";
+import { useState } from "react";
+import { AxiosError } from "axios";
+import { useMutation } from "@tanstack/react-query";
+
+import { deleteLikeProfile, postLikeProfile } from "@/app/_common/api/profile";
+
+import { ResponseError } from "@/app/_common/types/response";
+
+interface ToggleLike {
+  isLiked: boolean;
+  likeInfo: { senderId: number; receiverId: number };
```

Comment:
```suggestion
  isLiked: boolean;
  likeInfo: ProfileLikeBody;
```

이 부분 `ProfileLikeBody`로 변경하면 유지보수가 더 편해질 것 같네요.